src/GovUk.Education.ExploreEducationStatistics.Data.Api/Controllers/TableBuilderController.cs

```diff
@@ -18,20 +17,7 @@ public TableBuilderController(IDataService<TableBuilderResultViewModel> dataServ
         }
 
         [HttpPost]
-        [Obsolete("TODO DFE-1277 Remove when table tool switches to new endpoint")]
-        public ActionResult<ResultViewModel> Query([FromBody] ObservationQueryContext query)
-        {
-            var tableResultViewModel = _dataService.Query(query);
-            return new ResultViewModel
-            {
-                Footnotes = tableResultViewModel.SubjectMeta.Footnotes,
-                TimePeriodRange = tableResultViewModel.SubjectMeta.TimePeriodRange,
-                Result = tableResultViewModel.Results
-            };
-        }
-
-        [HttpPost("new")]
```

Comment:
Merge in conjunction with
https://github.com/dfe-analytical-services/explore-education-statistics/pull/600